client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -62,7 +62,7 @@ class WaitingTwoFactorNotificationApproval extends Component {
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.pushSuccess && nextProps.pushSuccess ) {
 			const { translate } = this.props;
-			this.props.successNotice( translate( 'Verified successfully!' ) );
+			this.props.successNotice( translate( 'Logging in…' ) );
```

Comment:
We already have the dot dot dot version, which is obviously lame.  I'll fix that version so we can keep this one here as is :)